templates/microprofile-kumuluzee/files/README.md

```diff
@@ -1,14 +1,22 @@
-# KumuluzEE template for Platform.sh
+# KumuluzEE for Platform.sh
 
-This project provides a starter kit for KumuluzEE MicroProfile projects hosted on [Platform.sh](http://platform.sh).
+This project provides a starter kit for KumuluzEE Eclipse MicroProfile projects hosted on [Platform.sh](http://platform.sh).
```

Comment:
The self-link in the description is unnecessary.

Reply:
Thank you, I removed it from all README.md